packages/tx-construction/src/tx-builder/TxBuilder.ts

```diff
@@ -179,6 +183,7 @@ export class GenericTxBuilder implements TxBuilder {
             return {
               ctx: {
                 auxiliaryData,
+                handles: [],
```

Comment:
I have added the handles here, but I'm unsure how outputBuilder and txBuilder resolve the final response.. and I think that this needs to be added in more places.

Reply:
@mkazlauskas can you please pair up with @VanessaPC and flesh out the implementation detail?

Reply:
@mkazlauskas can you please pair up with @VanessaPC and flesh out the implementation detail?